.github/workflows/push.yml

```diff
@@ -0,0 +1,118 @@
+name: push-pipeline
```

Comment:
I think this workflow is repetitive to build only image push is difference, So i think we can merge the both and use if condition for push job.
Reference: https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions

Reply:
I think it is better to make each pipeline in a separate YAM. It is easy to refactor and add different types of integration to the different pipelines.
Also, we are following the same process in all the repo.